Library/SunPhase.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 import CoreLocation
 import Suntimes
 
-enum SunPhase {
+enum SunPhase: Int {
```

Comment:
It's for testing, when you want to give back a random SunPhase its convenient to have this as Int. And it don't hurt does it?